integ-test/src/test/java/org/opensearch/sql/correctness/runner/connection/JDBCConnection.java

```diff
@@ -17,6 +17,7 @@
 import java.sql.Statement;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
```

Comment:
a mistake?

Reply:
Yes, good catch, thanks. Fixed in b5fd1208eb4d4012ce8a7fa16b93a2b50ac4eaf1